packages/cli/src/controllers/invitation.controller.ts

```diff
@@ -18,6 +18,7 @@ import type { User } from '@/databases/entities/User';
 import validator from 'validator';
 
 @Service()
+@Authorized()
```

Comment:
This could be changed in the future. There are some owner checks I didn't change since they're around instance setup (the `/owner` API). At the moment I'm relying on `@Authorized` make sure the passport middleware is being run. adi mentioned that we should make all the rest controllers authenticated by default, which would solve this issue as well. So I think maybe adding a new decorator called something like `@IsInstanceOwner` or similar could solve my issue and then making them all require auth would be the way forward?